hera_cal/vis_clean.py

```diff
@@ -398,16 +488,30 @@ def vis_clean(self, keys=None, x=None, data=None, flags=None, wgts=None,
                 self.fourier_filter(keys=[k], filter_centers=filter_centers, filter_half_widths=filter_half_widths,
                                     mode=mode, suppression_factors=suppression_factors,
                                     x=x, data=data, flags=flags, wgts=wgts, output_prefix=output_prefix,
-                                    ax=ax, cache=cache, skip_wgt=skip_wgt, verbose=verbose, overwrite=overwrite, **filter_kwargs)
+                                    ax=ax, cache=cache, skip_wgt=skip_wgt, verbose=verbose, overwrite=overwrite,
+                                    skip_flagged_edge_freqs=skip_flagged_edge_freqs,
+                                    skip_flagged_edge_times=skip_flagged_edge_times,
+                                    skip_gaps_larger_then_filter_period=skip_gaps_larger_then_filter_period,
+                                    flag_filled=flag_filled,
+                                    **filter_kwargs)
             else:
                 self.fourier_filter(keys=[k], filter_centers=filter_centers, filter_half_widths=filter_half_widths,
                                     mode=mode, tol=tol, x=x, data=data, flags=flags, wgts=wgts, output_prefix=output_prefix,
-                                    ax=ax, skip_wgt=skip_wgt, verbose=verbose, overwrite=overwrite, **filter_kwargs)
+                                    ax=ax, skip_wgt=skip_wgt, verbose=verbose, overwrite=overwrite,
+                                    skip_flagged_edge_freqs=skip_flagged_edge_freqs,
+                                    skip_flagged_edge_times=skip_flagged_edge_times,
+                                    skip_gaps_larger_then_filter_period=skip_gaps_larger_then_filter_period,
+                                    flag_filled=flag_filled,
+                                    **filter_kwargs)
 
     def fourier_filter(self, filter_centers, filter_half_widths, mode,
                        x=None, keys=None, data=None, flags=None, wgts=None,
                        output_prefix='clean', zeropad=None, cache=None,
-                       ax='freq', skip_wgt=0.1, verbose=False, overwrite=False, **filter_kwargs):
+                       ax='freq', skip_wgt=0.1, verbose=False, overwrite=False,
+                       skip_flagged_edge_freqs=False, skip_flagged_edge_times=False,
+                       skip_gaps_larger_then_filter_period=False,
+                       flag_filled=False, clean_flags_in_resid_flags=False,
+                       **filter_kwargs):
```

Comment:
should also note in the method description that this is just a wrapper of `uvtools.dpsec.fourier_filter`

Reply:
It looks like this is mentioned at the beginning of the docstring (you may have added it)? Let me know if this is good.

Reply:
yeah its fine, i added it